src/main/kotlin/com/atlassian/performance/tools/report/api/FullReport.kt

```diff
@@ -42,6 +42,11 @@ class FullReport {
             results = results
         )
 
+        val jdkBenchmarkTest = results.any { result -> result.cohort.contains("jdk") }
```

Comment:
It seems it's something you'd like to do for a specific test. You can create your own judge and inline into the specific test. There's no need to add it to the library.

Reply:
As I mentioned in [the issue](https://ecosystem.atlassian.net/browse/JPERF-430), the standard `summary-per-cohort.csv` report achieves the same goal - a tabular summary across cohorts.

Reply:
yes, I see now :) going to close this PR, thanks!